alacritty/src/window_context.rs

```diff
@@ -109,7 +115,12 @@ impl WindowContext {
         // The PTY forks a process to run the shell on the slave side of the
         // pseudoterminal. A file descriptor for the master side is retained for
         // reading/writing to the shell.
-        let pty = tty::new(&pty_config, display.size_info.into(), display.window.x11_window_id())?;
+        let pty = tty::new(
+            &pty_config,
+            display.size_info.into(),
+            u64::from(display.window.id()),
```

Comment:
Do we need from here and not `into()`? I think the most common is `into`? Besides that we have 2 arguments doing the same thing...

Reply:
I guess that's a good point, I forgot for a second that we're using Winit's window ID now which is the same as the X11 window id.